File: anggaRizky/CounterApp/app/src/main/java/com/tutorial/counterapp/MainActivity.java
package com.tutorial.counterapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    Button btnCounter;
    TextView txtCounter;
    Integer i;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        i=0;
        btnCounter  = (Button) findViewById(R.id.btn_counter);
        txtCounter  = (TextView) findViewById(R.id.txt_counter);

        btnCounter.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                i++;
                txtCounter.setText(i +"");
            }
        });
    }
}

File: sekolahKoding/helloKoding/app/src/main/java/com/tutorial/hellokoding/MainActivity.kt
package com.tutorial.hellokoding

import android.content.res.Resources
import android.graphics.drawable.Drawable
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import androidx.core.content.res.ResourcesCompat
import androidx.databinding.DataBindingUtil
import com.tutorial.hellokoding.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
//        setContentView(R.layout.activity_main)
          binding = DataBindingUtil.setContentView(this, R.layout.activity_main)
        //membuat variabel
//        val btnClickMe = findViewById(R.id.clickMe) as Button

        binding.clickMe .setOnClickListener {
            val name = binding.etInputName.text.toString().trim()
            val imgRes = resources.getIdentifier(name,"drawable",packageName)
            if(name == null || name.trim() == "")
                Toast.makeText(this@MainActivity,"anda harus memasukan nama ",Toast.LENGTH_SHORT).show()
            else if(name.equals(imgRes)){
                binding.tvGreeting.setText(name)
                //change image
                Toast.makeText(this@MainActivity,"nama logo tidak tersedia!",Toast.LENGTH_SHORT).show()
                val imgRes = resources.getIdentifier("athafariz","drawable",packageName)
                binding.ivLogo.setImageResource(imgRes)
            }else{
                binding.tvGreeting.setText(name)
                //change image
                binding.ivLogo.setImageResource(imgRes)
                Toast.makeText(this@MainActivity,"tersedia!",Toast.LENGTH_SHORT).show()
            }
        }

    }
}
